### PR TITLE
Add filter buttons for zero style (empty, slash, dot)

### PR DESCRIPTION
**Pull request by jgreely**
I'm a slashed-zero kind of guy, so I thought I'd add the option to filter the font list to handle the common types.

I'm not happy with how the new buttons change the filter box layout, but I didn't have much luck mucking with your CSS.

### CODE REVIEW

**Comment by braver** [comment]
Yeah, I'll probably have to re-arrange some stuff to make space, or change the interaction a bit. Thanks for the contribution, especially the effort to update the data file. 

We also need to think about what to do with fonts like Input that actually let you choose different styles for the zero before you download it. And there are actually quite a bunch that will have an open type alternative for it so you can select either slashed, dotted or empty. I always just display the default now, probably most editors or IDE's don't let you set open type options anyway, but those alternatives actually make things a bit complicated. 

The zero style is probably the most important one though, so we should try to make that work correctly for all fonts that have more than one option here. At the very least the value in fonts.json should be an array. And then the implementation becomes the next problem.

**Comment by braver** [comment]
I have a functioning implementation going in #85 based on your work. So that's great. And I think maybe I should just release that. I mean, it's true that some fonts with dotted zeroes can be switched to slashed zeroes via open type alternatives. But usually the ligatures usually can be disabled as well, and there can be even more subtle alternatives for a given font. Right now none of that is surfaced to users anyway. And it kind of depends on the environment where the font is used if any of these alternatives are available anyway. Don't even begin with all the weights in some fonts etc. 🤯 

Currently I'm focused on, "if you download this font, what do you get?" and that's not bad. I'm thinking what needs to happen to really fill out all the details, is to fill all the description properties for all the fonts and display that somewhere. Ah, there is always more work to be done. I should get paid for this 😆 

Anyway, tldr; thanks so much for your contribution. If you want to check it out, see #85. I'll sleep on it and then probably put it live. 🎉
